advocacy_docs/partner_docs/CommVaultGuide/06-CertificationEnvironment.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: 'Certification Environment'
```

Comment:
Lets just use

Certification date
EDB Postgres - versions
Commvault - version

The rest we can remove.